compat.sql

```diff
@@ -4,9 +4,11 @@ ADD COLUMN IF NOT EXISTS attributes JSONB GENERATED ALWAYS AS (attributes_txt::j
 ALTER TABLE extrinsics
 ADD COLUMN IF NOT EXISTS params JSONB GENERATED ALWAYS AS (params_txt::jsonb) STORED NULL;
 
+DROP INDEX IF EXISTS data_block_datetime;
+
 CREATE UNIQUE INDEX IF NOT EXISTS data_block_id ON blocks (block_id);
 CREATE UNIQUE INDEX IF NOT EXISTS data_block_hash ON blocks (hash);
-CREATE INDEX IF NOT EXISTS data_block_datetime ON blocks (datetime);
+CREATE INDEX IF NOT EXISTS data_block_datetime ON blocks (((datetime)::timestamp(0) without time zone));
```

Comment:
please give this one a different name so it's not getting dropped and re-created on every update, something like `data_block_datetime_timestamp`

Reply:
Done